### ISSUE TITLE
loss function for multi-class

### ISSUE THREAD

**Issue by BKP05** [davidsbatista/ConvNets-for-sentence-classification#1]
In your stanford-sentiment classification code, you still using binary crossentropy as your loss function. But, I am not sure if this is the right approach. Don't you think that you need to use categorical crossentropy?

**Comment by davidsbatista** [comment]
You're right, I just updated the code.
